### PR TITLE
Update Tuya Integration markdown file.

### PR DESCRIPTION
**Pull request by zlinoliver**
## Proposed change

Tuya Smart always strive to provide the best development experience possible and keep updating our Home Assistant integration capabilities. Since the old version of Tuya Home Assistant integration supports limited categories and control commands, now we bring you our new integration with extended support for more categories and commands.

Our new integration is built on Tuya’s standard [Open APIs](https://developer.tuya.com/en/docs/cloud) and tackles the limits of categories and accessible control commands. We deliver more development-friendly features including virtual device debugging, official authorization, and technical supports, which will help you efficiently work on development with the Tuya ecosystem.

The existing [Tuya Integration](https://www.home-assistant.io/integrations/tuya/) is no longer maintained by Tuya officially, and will be deprecated. The estimated deprecation date is 2021.12.31. The new integration will be officially maintained by the Tuya Developer Team, for more information of the migration, please check this documentation [How to Migrate the Tuya Home Assistant Integration from the Old Version](https://developer.tuya.com/en/docs/iot/migrate-from-an-older-version?id=Kamee9wtbd00b)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This is the current documentation for the current Tuya integration in the `tuya` domain. This PR does not match the current release.

I'm not sure what you are trying to achieve with this PR?

File: source/_integrations/tuya.markdown [hunk 1]
```diff
@@ -2,102 +2,192 @@
...
-  - Cover
-  - Fan
-  - Light
-  - Scene
-  - Switch
+  - Large Home Appliances
+  - Small Home Appliances
+  - Electrical
+  - Lighting
+  - Security&Sensors
```
Comment (frenck) [inline source/_integrations/tuya.markdown]: These are not valid categories

File: source/_integrations/tuya.markdown [hunk 2]
```diff
@@ -2,102 +2,192 @@
...
-The `tuya` integration is the main integration to integrate [Tuya Smart](https://www.tuya.com) related platforms, except the Zigbee hub. This includes devices linked with the Tuya, Smart Life, and Jinvoo Smart apps. You will need your account information (username, password and account country code) from one of these apps to discover and control devices which related to your account.
 
-**Important**: Not all Tuya devices are supported by the `tuya API` used by this integration. For more details refer to [TuyaHA Library](https://github.com/PaulAnnekov/tuyaha).
+The Tuya Home Assistant integration is developed for controlling **Powered by Tuya (PBT)** devices using the [tuya-iot-python-sdk](https://github.com/tuya/tuya-iot-python-sdk) (a python version of [Tuya Open API](https://developer.tuya.com/en/docs/iot/api-reference?id=Ka7qb7vhber64)), which is officially maintained by the Tuya Developer Team.
+
+It applies to smart devices that you have added to the Tuya Smart or Smart Life app.
+
+## Supported device categories
+
+<style>
```
Comment (frenck) [inline source/_integrations/tuya.markdown]: Don't use inline HTML in our documentation.

**Comment by zlinoliver** [comment]
> This is the current documentation for the current Tuya integration in the `tuya` domain. This PR does not match the current release.
> 
> I'm not sure what you are trying to achieve with this PR?

Hi @frenck , this documentation PR is for the new Tuya integration, it's related to this PR: https://github.com/home-assistant/core/pull/53510. Should we use a new tuya v2 branch for the PR?

**Comment by zlinoliver** [comment]
We create a new PR for the tuya v2 integration release, please help to review this instead: https://github.com/home-assistant/home-assistant.io/pull/19519

**Pull request closed without merging**